plaid/time.go

```diff
@@ -0,0 +1,75 @@
+package plaid
+
+import (
+	"fmt"
+	"time"
+)
+
+// DateFormat defines the date format used for client-facing date strings.
+const DateFormat = "2006-01-02"
+
+// DateTimeFormat defines the datetime format used for client-facing date
+// strings.
+const DateTimeFormat = time.RFC3339
+
+// Date is a time.Time object whose default string format (including for json
+// serialization) is a client-facing date string.
+type Date time.Time
```

Comment:
The standard practice is to use embedding here.

``` go
type Date struct { time.Time }
```

Reply:
@icholy can you provide a link to the standard?

Reply:
What?

Reply:
I was wondering if you could provide a link that says that `type Date struct { time.Time }` is the way it should be done.

Reply:
I don't agree that it should be embedded in an anonymous struct, that will prevent callers from utilizing the existing `time.Time` functions, unless I am misunderstanding what you mean exactly @icholy

Reply:
@ntindall When you create a new type (the way you're doing) it prevents callers from using any methods on `time.Time` without explicitly casting.